### ISSUE TITLE
regloss and loss not decline

### ISSUE THREAD

**Issue by Salary-only-17k** [davidsandberg/facenet#942]
On 20180402-114759, I trained 13 classes of facial softmax.py model. Accuracy soon reached 1.00, regloss and loss did not change and decline. I mainly modified the FC layer.please how to solve it?it is overfit?
![image](https://user-images.githubusercontent.com/26852480/50463280-c338d600-09c5-11e9-88d4-73cda2069a14.png)
